.github/workflows/docker-publish.yml

```diff
@@ -29,6 +29,12 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set up QEMU
```

Comment:
hi @BbolroC , I'm not experienced on buildx. In fact, I only noticed its existing today :)

It is very interesting the use of QEMU to build for architectures other than the host OS.

Reply:
yep, lovely technique, but it would be unstable sometimes for complicated builds like cc-builder. I have introduced it because a procedure is simple (just building a go binary)